src/drivers/efi/Kconfig

```diff
@@ -6,3 +6,11 @@ config DRIVERS_EFI_VARIABLE_STORE
 	help
 	  Adds a driver that is able to read and write an EFI formatted
 	  VariableStore as used by tianocore.
+
+config DRIVERS_EFI_UPDATE_CAPSULES
+	bool "Include EFI update capsules driver"
+	depends on DRIVERS_EFI_VARIABLE_STORE && SMMSTORE_V2
```

Comment:
Isn't there any dependency on DRIVERS_EFI_FW_INFO from https://github.com/Dasharo/coreboot/pull/527 ?

I mean, capsules in EDK2 won't make sense if there is no ESRT/FMP, right?

Reply:
Rebased this PR on that one and updated.

https://github.com/Dasharo/coreboot/pull/509/commits/31467e195468dada6224dc7cd541d6f096a42494#diff-7ffa6040ba95cc7406e87024010ef25159653dc594f56d00205c9fe95825f3ff